spec/controllers/hyrax/stats_controller_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Hyrax::StatsController, type: :controller do
+  routes { HykuAddons::Engine.routes }
+
+  let(:user) { create(:admin) }
+  let(:account) { create(:account) }
```

Comment:
can we use `build_stubbed` or does this need `create`?

Reply:
For the `account` we can use `build_stubbed` but the users need to be records with their roles defined. I've updated the account however.